elasticsearch/elasticsearch.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Plugin Name: Elasticsearch
+ * Description: VIP Elasticsearch plugin
+ * Version:     0.1
+ * Author:      Automattic VIP
+ * Author URI:  http://wpvip.com
+ * License:     GPLv2 or later
+ * Text Domain: vip-search
+ * Domain Path: /lang/
+ *
+ * @package  elasticsearch
+ */
+
+namespace Automattic\VIP\Elasticsearch;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+/**
+* PSR-4-ish autoloading
+*
+* @since 1.0
+*/
+spl_autoload_register(
+	function( $class ) {
```

Comment:
For formatting, this can probably go on the line above (then we save one level of indention)

Reply:
Done.